```rootsql
43.1 Обработка исключений
# Когда происходит исключение, чтобы обойти его и выполнить код, вы можете использовать инструкцию try/except.
Блок try содержит код под сомнением, код который может вызвать исключение. 
В случае исключения выполнение кода в блоке try прерывается и передается коду в блоке except. 
Если не происходит никакой ошибки, код в блоке except не выполняется.
try:
    num1 = 7
    num2 = 0
    print (num1 / num2)
    print("Done calculation")
except ZeroDivisionError:
    print("An error occurred")
    print("due to zero division")
В приведенном выше коде, в инструкции except указан тип исключения (в нашем случае это ZeroDivisionError).
```
```rootsql
43.2 Обработка исключений
# Инструкция try может иметь несколько различных блоков except, чтобы обрабатывать различные исключения.
Блок except может содержать несколько исключений, которые указываются в круглых скобках; 
все они будут выполнены программой.
```
```rootsql
43.3 Обработка исключений
# Выражение except, не имеющее каких-либо определенных исключений, будет перехватывать все ошибки. 
Данное выражение следует использовать с осторожностью, 
так как оно может перехватывать какие-либо неожиданные ошибки и скрыть допущенные ошибки в коде.
# Уметь обрабатывать исключения особенно важно при работе с пользовательским вводом.
```